api/routes/participant.py

```diff
@@ -179,6 +179,7 @@ async def update_participant(
         'success': await participant_layer.update_single_participant(
             participant_id=participant_id,
             external_id=participant.external_id,
+            family_id=participant.family_id,
```

Comment:
This isn't plumbed up like this, and unfortunately not how the families are modelled. I'd probably recommend removing the familyID update and we can chat about how to update families separately.

Reply:
ok sounds good, I'll remove those changes and we'll see how we go with the sequence external_id update